spec/intl.html

```diff
@@ -1626,7 +1626,9 @@ <h1>Temporal.Calendar.prototype.dateFromFields ( _fields_ [ , _options_ ] )</h1>
               1. Let _result_ be ? ISODateFromFields(_fields_, _options_).
             1. Else,
               1. Let _overflow_ be ? ToTemporalOverflow(_options_).
-              1. Set _fields_ to ? PrepareTemporalFields(_fields_, « *"day"*, *"era"*, *"eraYear"*, *"month"*, *"monthCode"*, *"year"* », « *"day"* »).
+              1. Let _fieldNames_ be CalendarDateFields(_calendar_.[[Identifier]], « *"day"*, *"month"*, *"monthCode"*, *"year"* »).
+              1. Sort _fieldNames_ alphabetically.
```

Comment:
This might need more specification; to me this could mean the default .sort() just as well as it could mean `.sort((a, b) => a.localeCompare(b))`

Reply:
I figured this would not be sufficient, but as it's an open question whether we need this sorted at all, I preferred not to spend time researching what the correct language would be.

Reply:
I definitely think it does need to be sorted; ordering of things needs to be deterministic.

Reply:
It is already deterministic because it's "the order in which the calendar's `fields()` method returns them". Additionally for built-in calendars that's always under our control. The question is whether that's good enough for userland calendars, but determinism isn't the problem.

Reply:
ah, gotcha. it still seems like a useful invariant to force on userland calendars to me (that "fields" returns a consistent order)